README.md

```diff
@@ -33,11 +34,9 @@
 ## Getting started
 
 \*\*Describe how to use the software.\*\*
+Replace all the references of the <DATAPLANE_URL> & <WRITE_KEY> placeholder with actual values at (src/index.js)
+Follow https://help.shopify.com/en/manual/promoting-marketing/pixels/custom-pixels/manage?shpxid=6c5e6078-ECEC-4AC3-EE41-151B603831FB for adding the custom pixel in the store
 
 ## Contribute
```

Comment:
This section should also be removed given that we don't have this file.

Reply:
I think this is correct, it is pointing to the custom pixel code in the src folder here

Reply:
I think @saikumarrs meant Contribute section

Reply:
My comment is intended for the "Contribute" section.

Reply:
GitHub does not properly highlight the selected line 👎

Reply:
got it, updated